homeassistant/components/zamg/sensor.py

```diff
@@ -58,7 +54,7 @@
 class ZamgRequiredKeysMixin:
     """Mixin for required keys."""
 
-    col_heading: str
+    para_name: str
```

Comment:
I'd suggest to rename the attribute to `parameter_name` or just `parameter` unless `para_name` is a special concept in the API.